File: mobile/src/main/java/com/windscribe/mobile/robert/RobertSettingsPresenter.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.robert;

import android.content.Context;

public interface RobertSettingsPresenter {

    String getSavedLocale();

    void init();

    void onCustomRulesClick();

    void onDestroy();

    void onLearnMoreClick();

    void setTheme(Context context);
}

File: mobile/src/main/java/com/windscribe/mobile/robert/RobertSettingsPresenter.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.robert

import android.content.Context

interface RobertSettingsPresenter {
    val savedLocale: String?
    fun init()
    fun onCustomRulesClick()
    fun onDestroy()
    fun onLearnMoreClick()
    fun setTheme(context: Context)
}